tools/RepositoryManagement/DevHome.RepositoryManagement/DevHome.RepositoryManagement.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$(SolutionDir)ToolingVersions.props" />
+  <PropertyGroup>
+    <RootNamespace>DevHome.RepositoryManagement</RootNamespace>
+    <Platforms>x86;x64;arm64</Platforms>
+    <RuntimeIdentifiers>win-x86;win-x64;win-arm64</RuntimeIdentifiers>
+    <UseWinUI>true</UseWinUI>
+    <Configurations>Debug;Release;Debug_FailFast</Configurations>
```

Comment:
Curious: What does the new configuration do?

Reply:
Fail fast?  #3244.  FailFast is a diagnostic tool that allows developers to find exceptions that might normally have been caught when running in debug.